vochain/start.go

```diff
@@ -25,8 +26,21 @@ import (
 
 // NewVochain starts a node with an ABCI application
 func NewVochain(vochaincfg *config.VochainCfg, genesis []byte) *BaseApplication {
+	if vochaincfg.SnapshotURL != "" {
+		log.Infof("importing snapshot...")
+		snapshotPath, err := FetchFile(filepath.Join(vochaincfg.DataDir, "snapshot"), vochaincfg.SnapshotURL)
+		if err != nil {
+			log.Fatalf("cannot import snapshot: %s", err)
```

Comment:
FetchFile just downloads, so i guess this would be

"cannot download snapshot: %s"